javascript-timers/interval.js

```diff
@@ -0,0 +1,27 @@
+var interval;
+
+function countdown() {
+  interval = setInterval(text, 1000);
+}
+
+function text() {
+  var text = document.querySelector('h1');
+  if (text.textContent === '4') {
+    text.textContent = '3';
+  } else if (text.textContent === '3') {
+    text.textContent = '2';
+  } else if (text.textContent === '2') {
+    text.textContent = '1';
+  } else if (text.textContent === '1') {
```

Comment:
What if the countdown needed to start at 1000? You should try to find a way to do this without a conditional block for each number.